### ISSUE TITLE
More details needed for reproducibility on VQA

### ISSUE THREAD

**Issue by avi-jit** [xiaoman-zhang/PMC-VQA#4]
Thank you for releasing code and checkpoints to the new state-of-the-art VQA model! Could you please help us reproduce your results by providing some more details and clarifications?
1. How many epochs are the models trained for on PMC-VQA (and then on respective benchmarks)?
2. Which version of VQA-RAD is used? Most available sources cite 2248 QA pairs (including on [huggingface](https://huggingface.co/datasets/flaviagiammarino/vqa-rad)) but your paper cites 3515 QA pairs (which perhaps also includes the 1267 framed questions, as described on page 5 of the [original paper](https://www.nature.com/articles/sdata2018251.pdf).
4. Is the learning rate same (2e-5) for all models and all parameters, and also for different datasets (PMC-VQA and then the individual VQA benchmarks)?

**Comment by xiaoman-zhang** [comment]
1. 5 epochs
2. In our work, we followed the train/test split (https://vision.aioz.io/f/777a3737ee904924bf0d/?dl=1) used in previous works like PMC-CLIP, and ARL( https://github.com/zhjohnchan/ARL/issues/3).
3. Yes, we use 2e-5 as default for all models

**Comment by avi-jit** [comment]
Thank you so much! @xiaoman-zhang  These details really help but we are still unfortunately unable to reproduce the results so far due to a few assumptions we need to make about missing specifications on evaluation. Would it be possible for you to release VQA-RAD evaluation script (and perhaps the models finetuned on it) for us to compare against your state-of-the-art model?

**Comment by xiaoman-zhang** [comment]
We have updated the fine-tuning and evaluation code for the MedVInT-TD model on Slake and VQA-RAD.

**Comment by vilin777** [comment]
Could you please provide the Train_csv and Test_csv for Slake datasets  or the pre-processing code? Thank you.
